mapsforge-map-reader/src/main/java/org/mapsforge/map/reader/MapFile.java

```diff
@@ -237,6 +238,12 @@ public MapFile(File mapFile, String language) {
             this.databaseIndexCache = new IndexCache(this.inputChannel, INDEX_CACHE_SIZE);
 
             this.timestamp = mapFile.lastModified();
+
+            // 2024: This may be removed in the future when world.map users get used to the map priorities feature
+            if ("world.map".equalsIgnoreCase(mapFile.getName())) {
+                // Some random (very) negative value no one will use
+                setPriority(Integer.MIN_VALUE + 374);
+            }
```

Comment:
@Sublimis We should not do this inside the library.
This must be done by the apps that know the map content.

Also, the map names are not fixed, they can be changed freely.
For example [paws.cz](https://osm.paws.cz/) has europe_admin map.

Reply:
@Sublimis Let's put it only in the sample apps.
Android has it, we can also put it in desktop samples.

Reply:
@Sublimis We can set it manually on desktop samples as well, just like on Android.
They are just examples.

Reply:
It's actually better like this, without assuming that the world.map file should be present, so the user decides which files to use. Because desktop sample has no startup menu like the Android app, it's just a single app.